core/src/version.rs

```diff
@@ -17,16 +17,16 @@ pub const SEMVER: &str = env!("CARGO_PKG_VERSION");
 pub const BUILD_ID: &str = env!("LIBRESPOT_BUILD_ID");
 
 /// The protocol version of the Spotify desktop client.
-pub const SPOTIFY_VERSION: u64 = 117300517;
+pub const SPOTIFY_VERSION: u64 = 124200290;
```

Comment:
When updating the SPOTIFY_VERSION, I prefer to also update the protobuf tree so that it keeps in sync.
Not sure if you want to expand this PR to cover that scope (but feel free if you want). Can you use login5 on the previous SPOTIFY_VERSION?

Reply:
good point, just updated them because i saw new version numbers. will check if it might break the login5 mobile login

Reply:
Yeah login on mobile still works without the update of the versions. As does the access-token retrieval.

Reply:
Thanks.